docs/README.md

```diff
@@ -40,6 +40,7 @@ The `add-to-cart-button` is a block responsible for adding products in the [Mini
 | Prop name               | Type      | Description                                                                       | Default value        |
 | ----------------------- | --------- | --------------------------------------------------------------------------------- | -------------------- |
 | `onClickBehavior`       | `enum` | Controls what happens when users click on the button. Possible values are: `go-to-product-page`, `add-to-cart`, and `ensure-sku-selection` (if multiple SKUs are available, users will be redirected to the product page to select the desired one. If the product only has 1 SKU available, it will be added to the cart once the button is clicked on). | `add-to-cart`              |
+| `onClickEventPropagation`       | `enum` | Controls how the 'onClick' event triggered upon user clicks is propagated to other elements in the DOM. Possible values are: `disabled` and `enabled`. | `disabled`              |
```

Comment:
What `other elements in the DOM` means? What is DOM?

Reply:
The DOM is a representation of the tree-like structure of HTML pages. I guess a better way to put this would be
`...is propagated to parent elements`. 

What I'm trying to say is that, when an event happens on a certain element, in this case, a 'click' event happens in the `<button>` element, this event "bubbles up", and the parent elements can also receive and act on this click. Sometimes you don't want that to happen, that's why we allow you to set this prop to `disabled` or `enabled`. Does that make sense? 😂

Reply:
Hmmmm, I think I got it! Let me know what you think about my suggestion!